optaplanner-test/src/main/java/org/optaplanner/test/api/change/MockProblemChangeDirector.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2022 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.optaplanner.test.api.change;
```

Comment:
If we create this class, can we keep the suffix of the package equal to it's original? Except for the module name package part
Original:
  org.optaplanner.core.api.solver.change.ProblemChangeDirector
Test version:
  org.optaplanner.test.api.solver.change.MockProblemChangeDirector

```suggestion
package org.optaplanner.test.api.solver.change;
```